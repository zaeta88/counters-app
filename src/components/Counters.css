.content {
  background-color: #fff;
  border-radius: 3px;
}

.right-button {
  margin-right: 15px;
  padding: 5px 10px;
}

.title-bar {
  background-color: #fff;
  padding: 8px;
  border-radius: 3px;
}