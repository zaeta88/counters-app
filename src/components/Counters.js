import React, { Component } from 'react';
import './Counters.css';

class Counters extends Component {
  constructor(props) {
    super(props);
    this.state = {
    };
  }

  render() {
    return (
      <div></div>
    );
  }
}

export default Counters;