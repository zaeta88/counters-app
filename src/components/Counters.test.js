import React from 'react';
import { shallow } from 'enzyme';
import Counters from './Counters';


describe('React.Component Counters', () => {
  it('should render correctly', () => {
    const component = shallow(<Counters handleActiveNav = { () => {} } />);
    expect(component).toMatchSnapshot();
  });
});
