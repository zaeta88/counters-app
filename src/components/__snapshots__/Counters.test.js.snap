// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React.Component Counters should render correctly 1`] = `ShallowWrapper {}`;
